### PR TITLE
Update AdministratorCommands.pwn for /show discord + Removed /ts 

### PR DESCRIPTION
**Pull request by Specifer**
Implementing #544 /show discord

### CODE REVIEW

File: pawn/Elements/AdministratorCommands.pwn [hunk 1]
```diff
@@ -1058,7 +1063,7 @@ lvp_show(playerId, params[]) {
     }
 
 ShowHelp:
-    SendClientMessage(playerId, Color::Information, "Usage: /show [beg/caps/donate/forum/interior/nick/reg/report/rules/ship/spam/swear/ts/weaps]");
+    SendClientMessage(playerId, Color::Information, "Usage: /show [beg/caps/donate/forum/interior/nick/reg/report/rules/ship/spam/swear/discord/weaps]");
```
Comment (RussellLVP) [inline pawn/Elements/AdministratorCommands.pwn]: Please keep this list alphabetized.

File: pawn/Elements/AdministratorCommands.pwn [hunk 2]
```diff
@@ -972,6 +972,11 @@ lvp_show(playerId, params[]) {
         showInfo = true;
         format(g_message, sizeof(g_message), "Save money and stats! Register your nickname at www.sa-mp.nl");
     }
+    
+    else if (!strcmp(showParameter, "discord", true, 4)) {
+        showInfo = true;
+        format(g_message, sizeof(g_message), "Join us on Official LVP Discord Server at www.discord.sa-mp.nl ");
```
Comment (RussellLVP) [inline pawn/Elements/AdministratorCommands.pwn]: 1. Official is not spelled with a capital,
  1. It's discord.sa-mp.nl, no use of "www." in front it it,
  1. No space at the end.

"Join us on the official LVP Discord server at discord.sa-mp.nl"

**Review by RussellLVP — CHANGES_REQUESTED** [review]

File: pawn/Elements/AdministratorCommands.pwn [hunk 3]
```diff
@@ -975,7 +975,7 @@ lvp_show(playerId, params[]) {
     
     else if (!strcmp(showParameter, "discord", true, 4)) {
         showInfo = true;
-        format(g_message, sizeof(g_message), "Join us on Official LVP Discord Server at www.discord.sa-mp.nl ");
```
Comment (Specifer) [inline pawn/Elements/AdministratorCommands.pwn]: @RussellLVP , Changes that had been requested has been made !

File: pawn/Elements/AdministratorCommands.pwn [hunk 4]
```diff
@@ -1063,7 +1063,7 @@ lvp_show(playerId, params[]) {
     }
 
 ShowHelp:
-    SendClientMessage(playerId, Color::Information, "Usage: /show [beg/caps/donate/forum/interior/nick/reg/report/rules/ship/spam/swear/discord/weaps]");
+    SendClientMessage(playerId, Color::Information, "Usage: /show [beg/caps/discord/donate/forum/interior/nick/reg/report/rules/ship/spam/swear/weaps]");
```
Comment (Specifer) [inline pawn/Elements/AdministratorCommands.pwn]: @RussellLVP , Changes that had been requested has been made !

File: pawn/Elements/AdministratorCommands.pwn [hunk 5]
```diff
@@ -972,6 +972,11 @@ lvp_show(playerId, params[]) {
         showInfo = true;
         format(g_message, sizeof(g_message), "Save money and stats! Register your nickname at www.sa-mp.nl");
     }
+    
+    else if (!strcmp(showParameter, "discord", true, 4)) {
```
Comment (RussellLVP) [inline pawn/Elements/AdministratorCommands.pwn]: The "4" in this command needs to be a "7", as it reflects the length of the string that has to be matched.

`/show discovery` will now also show this message, which isn't intentional.

File: pawn/Elements/AdministratorCommands.pwn [hunk 6]
```diff
@@ -972,6 +972,11 @@ lvp_show(playerId, params[]) {
         showInfo = true;
         format(g_message, sizeof(g_message), "Save money and stats! Register your nickname at www.sa-mp.nl");
     }
+    
+    else if (!strcmp(showParameter, "discord", true, 7)) {
```
Comment (Specifer) [inline pawn/Elements/AdministratorCommands.pwn:888]: @RussellLVP , updated here as well as in /guide request too . Feel free to use that as it has these script lines too

**Comment by RussellLVP** [comment]
Thank you for your contribution, it will go live with LVP 44!

**Pull request merged**